src/TLabel.cpp

```diff
@@ -105,6 +107,12 @@ void TLabel::mouseDoubleClickEvent(QMouseEvent* event)
 
 void TLabel::mouseReleaseEvent(QMouseEvent* event)
 {
+    auto labelParent = dynamic_cast<TConsole*>(parent());
+    if (labelParent && labelParent->mpDockWidget && labelParent->mpDockWidget->isFloating()) {
```

Comment:
Why `labelParent->mpDockWidget->isFloating()` ? Don't you still need the focus to be sent back to the main console for a profile when the label is in a *docked* user window - especially if multi-playing and there thus may be *multiple* main command lines visible?

*Note, this also applies to the corresponding line in the `TTextEdit` changes.*

Reply:
I tested it and it seems that this is not an issue when the UserWindow is docked, meaning the main command line remains in focus even if I click on a Label/Console in a docked UserWindow.

Reply:
Okay, just colour me surprised...!